packages/fleather/lib/src/widgets/controller.dart

```diff
@@ -14,6 +14,13 @@ List<String> _insertionToggleableStyleKeys = [
   ParchmentAttribute.inlineCode.key,
 ];
 
+class RemoteSelection {
```

Comment:
I think it's enough for now but we can have a builder function to build overlays with cursors too.

Reply:
yeah this will come.